<launch>
	<!-- global rosparam -->
	<param name="use_sim_time" value="true"/>

	<!-- rosbag -->
	<node pkg="rosbag" type="play" name="player" args="--clock $(env HOME)/rosbag/sq2_velodyne/dkan_outdoor/2018-10-31-08-37-11.bag -r 10"/>

	<!-- rviz -->
	<node pkg="rviz" type="rviz" name="rviz" args="-d $(find msg_conversion)/rviz_config/demo.rviz"/>

	<!-- pre-process -->
	<node pkg="msg_conversion" type="odometry_to_pose" name="odometry_to_pose">
		<param name="publish_frame" type="string" value="map"/>
	</node>

	<!-- main -->
	<node pkg="msg_conversion" type="pose_to_tf" name="pose_to_tf" output="screen">
		<param name="publish_frame" type="string" value="map"/>
		<remap from="/pose" to="/odom/to_pose"/>
	</node>

</launch>
